test/cbmc/include/callback_stubs.h

```diff
@@ -1,5 +1,5 @@
 /*
- * coreHTTP v2.1.0
+ * coreHTTP v3.0.0
  * Copyright (C) 2020 Amazon.com, Inc. or its affiliates.  All Rights Reserved.
```

Comment:
no copyright bump in cbmc proofs?

Reply:
So I had been told to just update it for the source files, but I'd have no issue with also updating the CBMC proofs and test files as well.